compiler/damlc/daml-prim-src/DA/Exception/GeneralError.daml

```diff
@@ -0,0 +1,24 @@
+-- Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+-- SPDX-License-Identifier: Apache-2.0
+
+{-# LANGUAGE CPP #-}
+{-# LANGUAGE NoImplicitPrelude #-}
+
+#ifndef DAML_EXCEPTIONS
+
+-- | HIDE
+module DA.Exception.GeneralError where
+
+import GHC.Types ()
+
+#else
+
+-- | MOVE DA.Exception
+module DA.Exception.GeneralError where
+
+import GHC.Types (Text)
+
+-- | Exception raised by `error`.
```

Comment:
we should probably also revisit the name of this soonish. I think `GeneralError` sounds a bit too general (no pun intended) for something only thrown by calls to `error`.

Reply:
What do you suggest? Maybe `Abort` or something, since it's also raised by `abort`?

Reply:
Let's discuss during the meeting today :)